app/build.gradle

```diff
@@ -11,6 +11,24 @@ plugins {
     id 'application'
     id 'jacoco'
     id 'com.diffplug.spotless' version '6.11.0'
+    id 'org.cyclonedx.bom' version '1.7.2'
+}
+cyclonedxBom {
+
+    // Specified the type of project being built. Defaults to 'library'
+    projectType = "application"
+
+    // Boms destination directory (defaults to build/reports)
+    destination = file("reports/sbom")
```

Comment:
I wouldn't overwrite this default.  I think it should go into the `build` directory with all of Gradle's other artifacts.  Then there isn't a need for the `README.md` either.

Reply:
Sounds fine to me. I mainly created the Readme.md just to register the directory in github. I'll make those changes.